examples/logging_example2.py

```diff
@@ -1,4 +1,4 @@
-from sem import logging
+from sem import utils, logging
 import sem
 
 #######################
```

Comment:
i noticed that examples/logging_example2.py defines `ns_path = './examples/ns-3'`, whereas examples/logging_example.py defines `ns_path = ns-3'`, instead.
Please update examples/logging_example.py to `ns_path = './examples/ns-3'` as well.